autode/log/methods.py

```diff
@@ -0,0 +1,47 @@
+
+
+class ComputationalMethods:
+    """Generic computational methods section built by autodE"""
+
+    def __add__(self, other):
+        """Add another sentence to the methods if it's not already present"""
+        assert type(other) is str
+
+        if other not in self._list:
+            self._list.append(other)
+
+        return None
+
+    def __repr__(self):
+        return self.__str__()
+
+    def __str__(self):
+        """String of the computational methods used in this initialisation
+        of autodE prepended with the current version"""
+        import autode
+        autode_str = (f'All calculations were performed in autodE '
+                      f'v. {autode.__version__}. ')
+
+        return autode_str + " ".join(self._list)
+
+    def add(self, other):
+        """Add a string to the methods"""
+        return self.__add__(other)
+
+    def clear(self):
+        """Clear the current string"""
+        self._list = []
+        return None
+
+    def string(self):
```

Comment:
ew

Reply:
rather not have .string() and just have str() ?

Reply:
yeah, it just seems messy this way